superset/connectors/sqla/models.py

```diff
@@ -169,15 +169,19 @@ def get_time_filter(
         col = self.get_sqla_col(label="__time")
         l = []
         if start_dttm:
-            l.append(col >= text(self.dttm_sql_literal(start_dttm)))
+            l.append(
```

Comment:
If there is now configuration for inclusive/exclusive start points, should we have another `if` condition in this block supporting `col > <start time literal statement>`?

Reply:
@willbarrett per SIP-15 the end state is `[start, end)` i.e., this should not be configurable. The reason for the interim configuration is to allow users to preview their charts using the `[start, end)` time range endpoints before saving and thus persisting this information in the chart form data.

The `time_range_endpoints` form-data is merely there as a mechanism to provide both the current and future states simultaneously for a period which enabling an institution to manually fix charts as necessary.